kit/src/test/java/com/oracle/javafx/scenebuilder/kit/fxom/FXMLPropertiesDisablerTest.java

```diff
@@ -58,8 +58,8 @@ public void that_property_value_is_set_to_false_on_MacOS() throws Exception {
     public void that_property_value_is_not_modified_on_Windows() throws Exception {
         classUnderTest = new FXMLPropertiesDisabler(OS.WINDOWS);
         String fxmlText = readResourceText("ContainerWithMenuSystemMenuBarEnabled.fxml");
-        assertTrue("ensures that test resource is correct",
-                fxmlText.contains("<MenuBar useSystemMenuBar=\"true\" VBox.vgrow=\"NEVER\" fx:id=\"theMenuBar\">"));
+        assertTrue(fxmlText.contains("<MenuBar useSystemMenuBar=\"true\" VBox.vgrow=\"NEVER\" fx:id=\"theMenuBar\">"),
+                "ensures that test resource is correct");
```

Comment:
JUnit5 API has changed here, so that the test description is now the last method argument. In JUnit4 it was the first argument. This change applies to all assertX methods with a description.